src/user-guide/writing-workflows/scheduler.rst

```diff
@@ -66,7 +66,16 @@ Some workflow events have related configurations e.g. for setting the timeout.
    :Event Handler: `abort handlers`
 
    The scheduler shut down early with error status, due to a fatal error
-   condition or a configured timeout.
+   condition.
+
+   .. note::
+
+      Not to be confused with ``abort on <event>`` settings.
+      The ``abort`` event is not raised when any such configured event occurs.
```

Comment:
Maybe worth mentioning that the corresponding timeout event does get raised.

Reply:
I couldn't think of a way to phrase this in a way that didn't make the sentence more difficult to understand, so I've just left it